cfme/services/catalogs/catalog_item.py

```diff
@@ -194,10 +194,10 @@ def create(self):
         view = navigate_to(self, 'Add')
         # Need to provide the (optional) provider name to the form, not the object
         provider_name = None
-        provider_required_types = ['AnsibleTower', 'Orchestration']
-        if self.item_type in provider_required_types \
-                or self.provider_type in provider_required_types:
+        if self.item_type == 'Orchestration':
             provider_name = self.provider.name
+        if self.item_type == 'AnsibleTower':
```

Comment:
**Required**
`elif` and change the variable name here since its not just a string anymore. maybe `provider_formvalue`